### ISSUE TITLE
"Files not uploaded" error, not same cause as #704

### ISSUE THREAD

**Issue by edmeades-lbto** [projectsend/projectsend#716]
## Type

Please complete with an X between the brackets to indicate the type of submission.
This is a:

[X] Bug report

[] Feature request

### Description of the problem

Without any known changes to environment, ProjectSend began throwing "Files not uploaded" when we attempt to upload files. It seemed that some browsers had more success than others, but regardless it is not working smoothly. 

I upgraded PS to the latest version but that did not change anything.

The storage target is an NFS mounted volume. I checked the permissions from the host and file storage computer and all looks good.

I read through ticket #704, but the conclusion there does not apply; we are not running selinux at all.

### List the steps to reproduce the issue

1. Attempt to upload files

2. Upload appears to complete, but fails with "Failed" and "undefined" in the filename field.

### Environment

* ProjectSend version: 1070
* php version: 5.4.16
* MySQL version: 15.1 Distrib 5.5.52-MariaDB, for Linux (x86_64) using readline 5.1

### IMPORTANT

This error has been reported before, but the solution does not apply to my install.

**Comment by edmeades-lbto** [comment]
Addendum for clarity: It's definitely not a permissions issue like #710; as stated above the previous install stopped working with no apparent changes. Regardless I checked the file ownership and everything on both sides of NFS is owned by apache's UID.

**Comment by edmeades-lbto** [comment]
Fixed 1070; was an ownership issue on the ProjectSend directory itself. This does not explain why the previous version stopped working suddenly. Leaving open for a bit in case anyone has any insight on possible causes for the initial failure.

**Comment by ignacionelson** [comment]
Hello! Thanks for leaving your solution!
About the problem happening suddenly, maybe something changed on the server itself. Did you update the OS, php, apache or mysql for example? Did you change your users or groups?

**Comment by edmeades-lbto** [comment]
No, as I said above, there were no updates to the system at that time. Our first suspicion was that it was some emergent incompatibility with the users' browsers, but we can't substantiate that.

**Comment by alexmaury254** [comment]
I want to ask you the solution to this problem! I'm having this same issue but I don't know what to do! Thanks a lot for any help!

**Comment by ignacionelson** [comment]
@alexmaury254 have you checked if there is any message in the error log? Maybe something related to the file process-upload.php that is called via ajax as the files are uploading?

**Comment by alexmaury254** [comment]
@ignacionelson No! There's no errors or warnings in log related to that file.

**Comment by edmeades-lbto** [comment]
@alexmaury254 , in my case I had initially focused on the data storage directory, but the problem was caused by the upper level ProjectSend install directory ownership becoming incorrect. Check the PS documentation and make sure that the whole tree is correct.

Did you look through the other tickets that have similar errors?

**Comment by ignacionelson** [comment]
This issue is being closed as part of a bulk cleanup. ProjectSend has undergone significant changes since this was reported (including r1720 and r1994), and many older issues no longer apply to the current codebase.

If you are still experiencing this problem on the latest version, please open a new issue with updated details and we'll be happy to look into it. Thank you for your contribution!
